index.html

```diff
@@ -34,31 +34,20 @@
 
       .b-example-divider {
         width: 100%;
-        height: 3rem;
-        background-color: rgba(0, 0, 0, .1);
-        border: solid rgba(0, 0, 0, .15);
-        border-width: 1px 0;
-        box-shadow: inset 0 .5em 1.5em rgba(0, 0, 0, .1), inset 0 .125em .5em rgba(0, 0, 0, .15);
```

Comment:
why did you change this ? any reason please explain

Reply:
this was not needed hence i removed